### PR TITLE
Improve release notes script

### PR DESCRIPTION
**Pull request by tomcallahan**
The current release notes script does not handle area labels that
are not two parts, such as ":ml".  As these area labels are rare,
I have simply hard-coded a title for these area labels.  In
addition, the script will not explicitly call out instances where
multiple area labels are present on an issue.

### CODE REVIEW

File: dev-tools/es_release_notes.pl [hunk 1]
```diff
@@ -32,7 +32,7 @@
     ">enhancement", ">bug",           ">regression",  ">upgrade"
 );
 my %Ignore = map { $_ => 1 }
-    ( ">non-issue", ">refactoring", ">docs", ">test", ">test-failure", ":Core/Build" );
+    ( ">non-issue", ">refactoring", ">docs", ">test", ":Core/Build", ">test-failure" );
```
Comment (pcsanwald) [inline dev-tools/es_release_notes.pl]: is the ordering change necessary?

**Review by pcsanwald — APPROVED** [review]

File: dev-tools/es_release_notes.pl (hunk 1 above)
Comment (tomcallahan) [reply dev-tools/es_release_notes.pl]: No, I based changes off of what was in 6.3, which was missing the ">test-failure" label, and so I added it there.  When I changed to master I pulled the wrong change.  Fixed

**Comment by tomcallahan** [comment]
Thanks for the review @pcsanwald

**Comment by tomcallahan** [comment]
@elasticmachine test this please

**Comment by tomcallahan** [comment]
@elasticmachine retest this please

**Comment by tomcallahan** [comment]
@elasticmachine retest this please

**Comment by colings86** [comment]
I backported this to 6.x, 6.5 and 6.4 since these improvements are also required for the 6.x series release notes too

**Pull request merged**